File: Learn/OC/exceptioncontion/exceptioncontion/ViewController.m
//
//  ViewController.m
//  exceptioncontion
//
//  Created by WangChao on 11/7/14.
//  Copyright (c) 2014 WangChao. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    [self testFor];
//    [self testExcep];

}

-(void)testFor
{
    for (int i = 0; i<100; i++) {
    NSLog(@" i%d" ,i);
}
 
}

-(void)testExcep
{
    NSArray *array = [[NSArray alloc] init];
    
    NSObject *i = [array objectAtIndex:0];
    
    NSLog(@"%@", i);
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: Learn/Swift/LearnHello/LearnHello/ViewController.swift
//
//  ViewController.swift
//  LearnHello
//
//  Created by WangChao on 6/18/14.
//  Copyright (c) 2014 WangChao. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    
    @IBAction func buttonpress(sender: AnyObject) {
    }
    @IBOutlet var web:UIWebView
    
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        web.loadRequest(NSURLRequest(URL:NSURL(string:"http://www.baidu.com")))
        
        
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

